templates/simulate.html

```diff
@@ -42,59 +51,69 @@
         </style>
     </head>
     <body>
```

Comment:
check if the components inside the body are actually under it's definitions (tabs / spaces)
and overall spacing for clarity